pkgs/development/libraries/cxx-prettyprint/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  name = "cxx-prettyprint-${version}";
```

Comment:
Please change this to conform to the naming guidelines https://hydra.nixos.org/build/38657747/download/1/nixpkgs/manual.html#sec-package-naming
> If a package is not a release but a commit from a repository, then the version part of the name must be the date of that (fetched) commit. The date must be in "YYYY-MM-DD" format. Also append "unstable" to the name - e.g., "pkgname-unstable-2014-09-23".